src/Dfe.PlanTech.Domain/Core/Enums/Pages.cs

```diff
@@ -1,6 +1,9 @@
+
 namespace Dfe.PlanTech.Domain.Core.Enums;
 
-public enum Pages
+public static class Pages
 {
-    Landing
+    public const string Landing = "Landing";
```

Comment:
are magic strings the way to go here?

Reply:
Ideally no, but it needs to match the "slug" field in Contentful. To me, this is no different than an enum (as it's still just a typed string), except this allows us to better match standard URL naming conventions (e.g. hyphens).

Open for suggestions on improvements on this though, i.e. what should be done instead?